go/oasis-node/cmd/registry/node/node.go

```diff
@@ -124,27 +116,9 @@ func doInit(cmd *cobra.Command, args []string) { // nolint: gocyclo
 			os.Exit(1)
 		}
 		logger.Info("entity ID provided, assuming self-signed node registrations")
-
-		isSelfSigned = true
 	} else {
-		entity, entitySigner, err = cmdCommon.LoadEntitySigner()
-		if err != nil {
-			logger.Error("failed to load entity and its signer",
-				"err", err,
-			)
-			os.Exit(1)
-		}
-
-		entityID = entity.ID
-		isSelfSigned = !entity.AllowEntitySignedNodes
-		if viper.GetBool(CfgSelfSigned) {
-			isSelfSigned = true
-		}
-		if !cmdFlags.DebugDontBlameOasis() && !isSelfSigned {
-			logger.Error("entity signed node descriptors are prohibited")
-			os.Exit(1)
-		}
-		defer entitySigner.Reset()
+		logger.Error("missing --node.entity_id command-line argument")
```

Comment:
Minor nit, but it would be nicer if the check above was inverted and the above block was outdented (no `else` needed as the command would exit). E.g. fail if there's no entity ID specified, proceed otherwise.

Reply:
Done :)